### PR TITLE
ES: localize 'Containers'

### LINKED ISSUE #578: [Spanish] localize 'Containers'
Localize the `content/en/container.md` document to Spanish

### PR DESCRIPTION
**Pull request by josefloressv**
### Describe your changes
Localize the `content/en/container.md` document to Spanish

### Related issue number or link (ex: `resolves #issue-number`)
Resolves #578

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Comment by 92nqb** [comment]
Hi @josefloressv, thanks a lot for your contribution!

You must change the target branch from [main:cncf](https://github.com/cncf/glossary/tree/main/) to [cncf:dev-es](https://github.com/cncf/glossary/tree/dev-es)

Let me know if you need help :)

File: content/es/container.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+Antes de que los contenedores estuvieran disponibles, se necesitaban máquinas separadas para ejecutar aplicaciones.
+Cada máquina requeriría su propio sistema operativo, que requiere CPU, memoria y espacio en disco,
+todo para que una aplicación individual funcione.
+Además, el mantenimiento, la actualización y el inicio de un Sistema Operativo es otra fuente importante de trabajo.
+
+## ¿Cómo ayuda?
+
+Los contenedores comparten el mismo Sistema Operativo y sus recursos de máquina,
+distribuyen la sobrecarga de recursos del sistema operativo y crear un uso eficiente de la máquina física.
```
Comment (krol3) [inline content/es/container.md:25]:
```suggestion
distribuyen la sobrecarga de recursos del sistema operativo y crean un uso eficiente de la máquina física.
```

File: content/es/container.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+Esta capacidad solo es posible porque los contenedores normalmente no pueden interactuar entre sí.
+Esto permite ejecutar muchas más aplicaciones en la misma máquina física.
+
+Sin embargo, existen limitaciones.
+Dado que los contenedores comparten el mismo Sistema Operativo, los procesos pueden considerarse menos seguros que las alternativas.
```
Comment (krol3) [inline content/es/container.md:30]:
```suggestion
Dado que los contenedores comparten el mismo sistema operativo, los procesos pueden considerarse menos seguros que las alternativas.
```

File: content/es/container.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+Los contenedores también requieren límites en los recursos compartidos.
+Para garantizar los recursos, los administradores deben restringir y limitar el uso de la memoria y la CPU para que otras aplicaciones no funcionen mal.
```
Comment (krol3) [inline content/es/container.md:32]:
```suggestion
Para garantizar los recursos, los administradores deben restringir y limitar el uso de la memoria y la CPU para que otras aplicaciones no tengan un rendimiento deficiente.
```

**Review by krol3 — CHANGES_REQUESTED** [review]
Algunas sugerencias @josefloressv

**Comment by josefloressv** [comment]
> Hi @josefloressv, thanks a lot for your contribution!
> 
> You have to change the target branch from [main:cncf](https://github.com/cncf/glossary/tree/main/) to [cncf:dev-es](https://github.com/cncf/glossary/tree/dev-es)
> 
> Let me know if you need help :)

@92nqb I changed this PR to use [cncf:dev-es](https://github.com/cncf/glossary/tree/dev-es) but that brought another problem because I forked from main, I have applied rebase commands but I only forked the main branch for this repo. Do you know another way to resolve this?

**Comment by electrocucaracha** [comment]
> > Hi @josefloressv, thanks a lot for your contribution!
> > You have to change the target branch from [main:cncf](https://github.com/cncf/glossary/tree/main/) to [cncf:dev-es](https://github.com/cncf/glossary/tree/dev-es)
> > Let me know if you need help :)
> 
> @92nqb I changed this PR to use [cncf:dev-es](https://github.com/cncf/glossary/tree/dev-es) but that brought another problem because I forked from main, I have applied rebase commands but I only forked the main branch for this repo. Do you know another way to resolve this?

Utiliza el argumento de `cherry-pick` para mover un commit de un branch a otro

**Comment by josefloressv** [comment]
Gracias Victor, tendré que cerrar este PR y enviaré otro a la rama correcta.

**Pull request closed without merging**